### ISSUE TITLE
SHA256 gist

### ISSUE THREAD

**Issue by hfossli** [krzyzanowskim/CryptoSwift#519]
Hello Marcin!

I hope everything is well. Thank you for this great library and resource! Fantastic. 

I created a gist for sha256 from your framework so it is to just drag and drop sha256 if that is all someone needs.. I hope you don't mind and I hope I am complying to the license. I didn't know if I should add my MIT license or not. I thought it was maybe good idea in order to comply to 

> you must not claim that you wrote the original software.

and

> Altered source versions must be plainly marked as such, and must not be misrepresented as being the original software

Anyway. Let me know if you want it different.

**Comment by hfossli** [comment]
Whoops. Forgot the links!
Here's the [gist](https://gist.github.com/hfossli/bb7b1f2fe6e89c5dc6ae40dfd2be1122) and here's a project where I'm using it [EllipticCurveKeyPair](https://github.com/agens-no/EllipticCurveKeyPair).

**Comment by krzyzanowskim** [comment]
The thing with the license is that you should not re-licence CryptoSwift code with another license. That means you can licence your parts for the MIT licence, but the CryptoSwift licence stays under the CryptoSwift licence.

I never encourage this as it probably won't be updated to the latest versions as I continuously improve the performance.

**Comment by hfossli** [comment]
Allright :) Thank you for the input. 👍 I didn't mean to relicense. I thought maybe I had to add my own license in order to not have the source misrepresented as being the original software. I don't want to take credit for any of that code as it is way over my head. I have now removed the MIT license. :+1:

I understand you don't encourage gists because they get out of date, but sometimes people just want the simple solution if it just works. :)
